desktop/packages/linux/aur/ruffle-nightly-bin/PKGBUILD

```diff
@@ -8,7 +8,7 @@ url="https://ruffle.rs/"
 license=('Apache' 'MIT')
 depends=(zlib libxcb alsa-lib)
 provides=(ruffle)
-conflicts=(ruffle-git)
+conflicts=(ruffle)
```

Comment:
Reading the docs, there should be `ruffle`, but TBH I do not see any difference in behavior, as packages conflict with `/usr/bin/ruffle` anyway.

Reply:
Huh, it's weird to see that this package both provides, and conflicts with, `ruffle` - but it makes sense I guess, and the "unofficial" `ruffle-git` package has the same. So these names aren't package names then, but are in some other, more abstract namespace...?

Reply:
See https://wiki.archlinux.org/title/PKGBUILD#conflicts

> Note that conflicts are checked against pkgname as well as names specified in the provides array. Hence, if your package provides a foo feature, specifying foo in the conflicts array will cause a conflict between your package and all other packages that contain foo in their provides array (i.e., you do not need to specify all those conflicting package names in your conflicts array).